style.css

```diff
@@ -1074,6 +1074,10 @@ blockquote, body, dd, dl, dt, fieldset, figure, hr, html, iframe, legend, li, ol
 	border-radius: var(--normal-border-radius) !important;
 }
 
+.img-circular {
```

Comment:
I think that a better idea would be to change the name of this class to `.rounded` - not only images can have a border radius of 50%;

Reply:
ok